apps/civicsignalblog/src/payload/components/actions/index.tsx

```diff
@@ -34,6 +36,7 @@ function BeforeDashboard() {
           }
 
           const data = await response.json();
+          // eslint-disable-next-line no-undef
```

Comment:
What could potentially be undef?

Reply:
@kilemensi , `window` is undefined during the build process and would fail

Reply:
... then should you add `/* eslint-env browser */` at the top of the file to let eslint know this file will run in `browser` env rather than silencing the issue?